lib_eio_linux/eio_linux.mli

```diff
@@ -248,4 +248,29 @@ module Low_level : sig
   (** [getaddrinfo host] returns a list of IP addresses for [host]. [host] is either a domain name or
       an ipaddress. *)
 
+  module Process : sig
+    type t
+    (** A subprocess *)
+
+    val spawn :
+      ?env:Spawn.Env.t ->
+      ?cwd:Spawn.Working_dir.t ->
+      ?stdin:Unix.file_descr ->
+      ?stdout:Unix.file_descr ->
+      ?stderr:Unix.file_descr ->
+      sw:Switch.t ->
+      string ->
+      string list ->
+      t
```

Comment:
Should be made safe like the rest of the API (e.g. using `FD.t`).

Reply:
https://github.com/ocaml-multicore/eio/pull/435/commits/0685201dd2c0c19d65048e20b3823d83d0466112